modules/ROOT/partials/misc/inline-formatting-of-list-bullets.adoc

```diff
@@ -0,0 +1,15 @@
+== Inline formatting.
+
+List bullets and list numbers can be spot- or inline-formatted.
+
+If an end-user selects the contents of a list item and applies inline formatting, such as a change of color or change of font-size, this spot formatting is also applied to the list item’s associated bullet or number.
+
+Only the bullets or list numbers associated with the selection are inline-formatted.
+
+NOTE: PowerPaste does not, currently, support inline-formatted bullets or list numbers. Inline-formatting, as applied to the bullets or list numbers, is lost when such material is copied and pasted.
+
+If the selection is one list item (ie, if the selection runs from an particular `<li>` tag to its associated `</li>` tag), only the bullet or number associated with the single selected list item takes on the inline-formatting.
```

Comment:
> only the bullet or number associated with the single selected list item

I assume you mean here that only that list item's bullet or number takes on the formatting, as well as the contents of that list item. If that's what you mean, I feel this is a bit confusing as it reads like the only thing that gets formatted is the bullet/number, not the text.

Reply:
To quote my commit message:

> Edited multiple paragraphs to make explicit the relationship between spot-formatting a selection and the bullets or list numbers that will, consequently, take on the equivalent spot-formatting.